integration/fabric/events/chaincode/chaincode/assetTransfer/asset_transfer_events.go

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
shall we change the name of the file and the package to better reflect what we do? Maybe just events.